tools/doc/html.js

```diff
@@ -8,6 +8,15 @@ var typeParser = require('./type-parser.js');
 
 module.exports = toHTML;
 
+// customized heading without id attribute
+var renderer = new marked.Renderer();
```

Comment:
fwiw, I also used template strings in #6495, I don’t really think that’s a problem. And sorry for the merge conflict, btw, but that should be limited to the package.json.